app/views/assignments/edit/_topics.html.erb

```diff
@@ -1,15 +1,38 @@
 <h3>Topics for <%= @assignment_form.assignment.name %> assignment</h3><br>
+<style>
+.custom-button {
+    display: inline-block;
+    padding: 5px 5px;
+    background-color: white;
+    color: black;
+    text-decoration: none;
+    border: 1px solid black;
+    border-radius: 5px;
+    font: Arial;
+}
+</style>
+<script>
+  const enableBiddingCheckbox = document.getElementById('enable-bidding-checkbox');
+  const biddingButton = document.getElementById('bidding-button');
+
+  enableBiddingCheckbox.addEventListener('change', function() {
+    biddingButton.style.display = enableBiddingCheckbox.checked ? 'inline-block' : 'none';
+  });
+</script>
 <input name="assignment_form[assignment][allow_suggestions]" type="hidden" value="false"/>
 <%= check_box_tag('assignment_form[assignment][allow_suggestions]', 'true', @assignment_form.assignment.allow_suggestions) %>
 <%= label_tag('assignment_form[assignment][allow_suggestions]', 'Allow topic suggestions from students?') %>
 <br>
 <input name="assignment_form[assignment][is_intelligent]" type="hidden" value="false"/>
-<%= check_box_tag('assignment_form[assignment][is_intelligent]', 'true', @assignment_form.assignment.is_intelligent?)%>
+<%= check_box_tag('assignment_form[assignment][is_intelligent]', 'true', @assignment_form.assignment.is_intelligent?, id: 'enable-bidding-checkbox')%>
 <%= label_tag('assignment_form[assignment][is_intelligent]', 'Enable bidding for topics?') %>
 <img src="/assets/info.png" title="This feature allow students to &quot;bid&quot; for topics.
       Instructor must specify when topics are assigned, by going to the Due Dates tab and
       entering a due date for &quot;signup&quot;."/>
 <br>
+<%= button_to 'View Bidding Details', {:controller => 'lottery', :action => 'bidding_details', :id => @assignment_form.assignment.id}, 
```

Comment:
Again, "details" is not specific enough to give an idea of what will be shown.  Would "Show bids by priority" be better?  Also, please capitalize only the first word on the button, not all words ... less "shouting", and notice that's how Github does it.